assets/javascripts/kitten/components/buttons/button/button.js

```diff
@@ -108,8 +108,8 @@ const checkedCircleIconStyle = size => {
 
 const CheckedCircleIcon = styled(KittenCheckedCircleIcon)`
```

Comment:
Je propose de sortir les props lors de l'appel du composant. Ca évite d'avoir des `data-`.

```suggestion
const CheckedCircleIcon = styled(({ big, tiny, ...others }) => <KittenCheckedCircleIcon { ...others } />)`
```

Reply:
Ah oui, très bien vu !

Reply:
👍